src/components/NamespaceDropdown.tsx

```diff
@@ -20,30 +19,23 @@ export class NamespaceDropdown extends React.PureComponent<NamespaceListType, {}
     this.props.refresh();
   }
 
-  onSelectTypesafe = (value: string) => this.props.onSelect({ name: value });
-
-  onToggle = (isOpen: boolean) => (isOpen ? this.props.refresh() : undefined);
+  handleSelectNamespace = (namespace: string) => this.props.onSelect({ name: namespace });
 
   render() {
-    const disabled = this.props.disabled ? true : false;
```

Comment:
I actually like the feedback of the disabling of the menus. It tells me its loading something and I should "slow down" and not click things so fast. It also adds additional confirmation that things are loading and that what I currently see in the graph is out of date. For example, I select namespace foo and quickly namespace bar then quickly, "opps, I meant to select abc". OK, the graph I am looking at now - is it foo? bar? abc? Is it done loading?

Disabling helps me determine that things are loading, and also helps me avoid queuing up multiple changes while its previously loading a different selection.

I am -1 on this change - keep it with the behavior it was doing before is what I would prefer.

Reply:
I have to agree with @jmazzitelli , I am afraid of fast click issues (which *always* seem to happen) and don't mind the disable when refreshing.

Reply:
Despite that I don't agree with this, I did add it back in.
When the refresh rate is set to 5 seconds and it takes 3 seconds to refresh (and lock up the controls) you barely have any time to make a change. And for larger meshes 5 seconds will totally lock up your Kiali because if it takes more than 5 seconds you don't have a second chance to ever change your settings -- infinite loop.

Reply:
if it takes 3 seconds to refresh the graph, we've got bigger problems. If it is taking that long, we also should somehow not refresh every 5 seconds - sounds like we are pounding the server and I can't imagine the resources the server is taking.